libs/sdk-core/src/swap_in/swap.rs

```diff
@@ -299,9 +306,8 @@ impl BTCReceiveSwap {
         Ok(())
     }
 
-    async fn refresh_monitored_swaps(&self, tip: u32) -> Result<Vec<SwapInfo>> {
-        let to_check = self.list_monitored()?;
-        for s in to_check {
+    async fn refresh_swaps(&self, swaps: Vec<SwapInfo>, tip: u32) -> Result<Vec<SwapInfo>> {
+        for s in swaps {
```

Comment:
Since it's not refreshing the monitored swaps, it should also not return them in L232 below.

Instead it should collect the successfully refreshed swaps (returned in L312) and return them as a vector.

If the list is not used anywhere, it can also just return nothing.

Reply:
Done